torch/package/package_exporter.py

```diff
@@ -230,6 +244,17 @@ def __init__(
 
         self.patterns: Dict[GlobGroup, _PatternInfo] = {}
         self._unique_id = 0
+        self._selective_interns: Dict[str, [str]] = {}
+        self._inverse_default_selective_intern_list = {}
```

Comment:
What is this for?

Reply:
Oh yeah no longer need this, I think I was using the inverse mapping for something before.